go/libraries/doltcore/sqle/enginetest/dolt_queries_diff.go

```diff
@@ -713,8 +743,11 @@ var Dolt1DiffSystemTableScripts = []queries.ScriptTest{
 		},
 		Assertions: []queries.ScriptTestAssertion{
 			{
-				Query:    "SELECT to_pk1, to_pk2, from_pk1, from_pk2, diff_type from dolt_diff_t;",
-				Expected: []sql.Row{{"2", "2", nil, nil, "added"}},
+				Query: "SELECT to_pk1, to_pk2, from_pk1, from_pk2, diff_type from dolt_diff_t;",
+				Expected: []sql.Row{
+					{"2", "2", nil, nil, "added"},
+					{"1", "1", nil, nil, "added"},
```

Comment:
This output makes sense to me. We should update the test name on line 731 though – it currently states that the intent is to test that diff data stops after a PK change, but that's not what the test is testing anymore.